### ISSUE TITLE
deprecate `cims:AssociationUsed "No"` props (or even delete them?)

### ISSUE THREAD

**Issue by VladimirAlexiev** [Sveino/Inst4CIM-KG#113]
- https://github.com/Sveino/Inst4CIM-KG/issues/26 maps `cims:inverseRole` to the standard prop `owl:inverseOf`
- https://github.com/Sveino/Inst4CIM-KG/issues/50 discusses the required reasoning.
I thought `inverseOf` reasoning is required since otherwise how will SHACL and SPARQL writers know which direction of an inverse pair to use?

However, @Sveino explained that `cims:AssociationUsed` points out which prop of an inverse pair is actually used, eg:
```ttl
cim:ACDCConverterDCTerminal.DCConductingEquipment a owl:ObjectProperty, owl:FunctionalProperty ;
  rdfs:label "DCConductingEquipment" ;
  rdfs:comment "A DC converter terminal belong to an DC converter."@en ;
  cims:AssociationUsed "Yes" ; ###########
  cims:multiplicity cims:M:1..1 ;
  owl:inverseOf cim:ACDCConverter.DCTerminals ;
  rdfs:domain cim:ACDCConverterDCTerminal ;
  rdfs:range cim:ACDCConverter .

cim:ACDCConverter.DCTerminals a owl:ObjectProperty, owl:InverseFunctionalProperty ;
  rdfs:label "DCTerminals" ;
  rdfs:comment "A DC converter have DC converter terminals. A converter has two DC converter terminals."@en ;
  cims:AssociationUsed "No" ; #############
  cims:multiplicity cims:M:0..n ;
  owl:inverseOf cim:ACDCConverterDCTerminal.DCConductingEquipment ;
  rdfs:domain cim:ACDCConverter ;
  rdfs:range cim:ACDCConverterDCTerminal .
```
My proposal is to make this explicit by using standard props:
```ttl
cim:ACDCConverter.DCTerminals owl:deprecated true;
  skos:scopeNote "This property is not asserted in instance data. Use its owl:inverseOf property instead"@en
```
An even "stronger" approach would be to **omit the definition** of the "recessive" prop and declare only its name in the "dominant" prop:
```ttl
cim:ACDCConverterDCTerminal.DCConductingEquipment prov:inverse "ACDCConverterDCTerminal.DCConductingEquipment".
```
This is the PROV approach: https://www.w3.org/TR/prov-o/#inverse-names.
It has been adopted in DCAT (at least in part): https://w3c.github.io/dxwg/dcat/#inverse-properties, https://github.com/w3c/dxwg/issues/1336 .

However, we have rich info (`owl:InverseFunctionalProperty, cims:multiplicity`) that we would lose if we use a mere name.

**Comment by VladimirAlexiev** [comment]
@Sveino please state whether CIM data will use `inverseOf` inference
- if not then `cims:AssociationUsed` holds precious info: which direction is actually present in the data.
(and this issue suggests to express this in a standard way)
- if yes then `cims:AssociationUsed` is less important

> can confuse with the meaning of deprecate we use in CIM = being it is still used, but it will be deleted in next or following versions

- skos:scopeNote clarifies this
- we could use two subprops of `owl:deprected`:
  - `cims:deprecatedWillBeDeletedInFuture`
  - `cims:deprecatedBecauseNotUsedInData`

**Comment by Sveino** [comment]
I agree with Chavdar. We need to `owl:inversOf`and we expect that there will be reasoning to derive the association that is not included in the instance file.  `cims:AssociationUsed` is a profile information. It will not be included in the vocabulary. It can therefore be translated into relevant SHACL validation without being included as RDFS.

**Comment by VladimirAlexiev** [comment]
Decided that:
- SHACL will continue to operate only over AssocitionUsed "Yes"
- It is possible to check in SHACL that the "wrong" (recessive) direction is not present: 
  - use a `sh:not [sh:property [sh:path <recessive prop>]]`
  - but should put them in a separate SHACL file: to be used on file validation only, but not repository validation
- inverse reasoning is useful, in particular for querying
- but won't be demanded in the standard because that's a competitive product feature.
